crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -73,7 +73,7 @@ fn pbr_input_from_standard_material(
     let NdotV = max(dot(pbr_input.N, pbr_input.V), 0.0001);
 
 #ifdef VERTEX_UVS
-    var uv = in.uv;
+    var uv = (pbr_bindings::material.uv_transform * vec3(in.uv, 1.0)).xy;
```

Comment:
I'm not sure if this should come before or after the tangent stuff. Does it matter?

Reply:
Since the transformed  `uv` is used in `parallaxed_uv` I think this should stay here.